CPS-0010/README.md

```diff
@@ -0,0 +1,276 @@
+---
+CPS: 10
+Title: Wallet Connectors
+Status: Open
+Category: Wallets
+Authors:
+    - Ryan Williams <ryan.williams@iohk.io>
+    - Vladimir Kalnitsky <vladimir@mlabs.city>
+Proposed Solutions: []
```

Comment:
For my own understanding, is this where future adopted Web-Wallet bridges like CIP-90 would go when merged?

Reply:
Yes, I believe this to be the intension.

from CIP-9999:

> A list of CIPs addressing the problem, if any

Reply:
thanks @Ryun1 then I assume for this same reason that CIP-30 _does not_ go here... due to its being the _problem_ rather than the _solution_ 😅

Reply:
> CIP-30 does not go here... due to its being the problem rather than the solution 😅

Yup! 😆
and currently I dont think CIP90 or CIP45 could go here either, as they do not fully satisfy the problem(s) described

Reply:
yes, all these CIPs seem to have a large factor in common 😆